### ISSUE TITLE
Bug: refresh button always seem to always throw error

### ISSUE THREAD

**Issue by VikingScientist** [TheBB/Geomaker#23]
The following is thrown into the terminal when I push the <kbd>Refresh</kbd> button 
```
Traceback (most recent call last):
  File "/home/kjetijo/.local/lib/python3.6/site-packages/GeoMaker-0.1-py3.6.egg/geomaker/asynchronous.py", line 343, in process
    retval = self.func(*args, **kwargs, manager=manager)
  File "/home/kjetijo/.local/lib/python3.6/site-packages/GeoMaker-0.1-py3.6.egg/geomaker/db.py", line 603, in _download_data
    with ZipFile(responsedata, 'r') as z:
  File "/usr/lib/python3.6/zipfile.py", line 1131, in __init__
    self._RealGetContents()
  File "/usr/lib/python3.6/zipfile.py", line 1194, in _RealGetContents
    endrec = _EndRecData(fp)
  File "/usr/lib/python3.6/zipfile.py", line 264, in _EndRecData
    fpin.seek(0, 2)
AttributeError: 'NoneType' object has no attribute 'seek'
```

Also I doesn't seem to actually download the data, even after waiting for quite some time.

**Comment by TheBB** [comment]
This happens when HTTP responds with non-OK. (Side note: display an actual error message.)

Can you go in the database and print the info on the jobs you are trying to download?

    sqlite3 -line ~/.local/share/geomaker/db.sqlite3 'select * from job;'

**Comment by TheBB** [comment]
After https://github.com/TheBB/Geomaker/pull/25 you should now get an error message with (hopefully) more information.

**Comment by VikingScientist** [comment]
![Screenshot from 2020-02-17 15-41-14](https://user-images.githubusercontent.com/3500376/74663412-fe82ff00-519b-11ea-9d3e-5cf0452d2dc7.png)

**Comment by VikingScientist** [comment]
I deleted the entire install by
`rm -rf ~/.local/share/geomaker`
and after this, there have been no new error. I don't know if this has removed all data from geomaker. All regions vanished from the GUI, but it still remembered my e-mail address when requesting downloads from kartverket.

**Comment by TheBB** [comment]
Maybe you had a job from more than two weeks ago still in the database. They remove the downloadable files after 14 days AFAIK.

**Comment by VikingScientist** [comment]
Doesn't seem to be working. After doing the delete operation outlined above I still don't get any new terrain data to download. No more error messages as popups or in the terminal, but I've waited 15 minutes for a 0.5km<sup>2</sup> region which seems excessive

**Comment by TheBB** [comment]
What's the job ID? (See code I posted eariler.) I started a job an hour ago and it's still running, so it might be something with hoydedata.no. Did you get an e-mail that says your job is ready? If not then I can only guess that it's really not ready.

**Comment by TheBB** [comment]
@VikingScientist mine just finished now, so maybe you can check yours.

**Comment by VikingScientist** [comment]
This one is gone now. The original error messages vanished after deleting the entire database. The "terrain data not appearing" was fixed after the discussion in #24, specifically [this](https://github.com/TheBB/Geomaker/issues/24#issuecomment-587439209) comment got a hotfix commit.
